documentation/tutorials/creating_synthetic_tokens_from_truffle_console.md

```diff
@@ -0,0 +1,151 @@
+ 
+1. Because we are a token sponsor for this synthetic token contract, we can redeem some of the tokens we minted even before the synthetic token expires. Let's redeem half.
+```js
+await syntheticToken.approve(emp.address,web3.utils.toWei("10000"))
+await emp.redeem({rawValue: web3.utils.toWei("50")})
+```
+ 
+2. Let’s check that our synthetic token balance has decreased and our collateral token balance has increased. 
+Our synthetic token balance should now be 50. 
+Because the contract does not have an on-chain price feed to determine the token redemption value for the tokens, it will give us collateral equal to the proportional value value of the total collateral deposited to back the 100 tokens (50/100 * 150 = 75). 
+Our collateral token balance should increase to 9,925. 
+```js
+(await collateralToken.balanceOf(accounts[0])).toString()
+// collateral token balance
+(await syntheticToken.balanceOf(accounts[0])).toString()
+// synthetic token balance
+await emp.positions(accounts[0])
+// position information
+```
+ 
+## Deposit and withdraw collateral
+ 
+1. As a token sponsor, we may wish to add additional collateral to our position to avoid being liquidated by a token holder. 
+Let’s deposit 10 additional collateral tokens to our position and see our updated balance, from 9,925 to 9,915. 
+```js
+await emp.deposit({rawValue: web3.utils.toWei("10")})
+(await collateralToken.balanceOf(accounts[0])).toString()
+```
+ 
+2. For a token sponsor to withdraw collateral from his position, there are typically 2 ways to do this. 
+Read this [explainer](../explainers/synthetic_tokens.md) for more information. 
+In this scenario, because we are the only token sponsor, we will have to withdraw collateral the “slow” way. First, we need to request a withdrawal of 10 collateral tokens.
+```js
+await emp.requestWithdrawal({rawValue: web3.utils.toWei("10")})
+```
+ 
+3. Now, we need to simulate the withdrawal liveness period passing without a dispute of our withdrawal request. 
+Because we set this period to be 1000 when we parameterized and deployed the contract, let’s simulate 1001 of time elapsing. 
+Once the 1001 has elapsed, we can withdraw the amount we previously requested. 
+```js
+await emp.setCurrentTime((await emp.getCurrentTime()).toNumber() + 1001)
+await emp.withdrawPassedRequest()
+```
+ 
+4. Let’s now check that our collateral token balance has returned to 9,925. 
+```js
+(await collateralToken.balanceOf(accounts[0])).toString()
+// collateral token balance
+```
+<!-- 
+
+--END OF TUTORIAL--
+Notes: We might prefer to show people how to settle a contract after expiration using a CLI tool so they can change between token sponsor, token holder, and DVM voter personas more easily. 
+
+This is particularly relevant for settling a token after the expiration time, when as a token sponsor they might settle, as a voter they might decide on the final token redemption value, and as token holder they might redeem. 
+-->
```

Comment:
Still not a huge fan of this notes section, but since it's smaller and more polished, I think that's fine.